modules/apps/asset/asset-categories-admin-web/src/main/resources/META-INF/resources/edit_asset_vocabulary.jsp

```diff
@@ -113,14 +113,16 @@ renderResponse.setTitle((vocabulary == null) ? LanguageUtil.get(request, "add-vo
 						'<liferay-ui:message key="are-you-sure-you-want-to-create-this-vocabulary-with-public-visibility" />';
 				}
 
-				if (
-					confirm(
+				Liferay.Util.openConfirm({
```

Comment:
`openConfirm` or `openConfirmModal`? I think that we don't declare `openConfirm` anywhere in the codebase.

Reply:
Yes, I forgot this case 😅. Thanks Kresso 🫂